dlp/snippets/src/main/java/dlp/snippets/InspectBigQuerySendToScc.java

```diff
@@ -112,8 +116,35 @@ public static void inspectBigQuerySendToScc(
               .build();
 
       // Send the job creation request and process the response.
-      DlpJob createdDlpJob = dlpServiceClient.createDlpJob(createDlpJobRequest);
-      System.out.println("Job created successfully: " + createdDlpJob.getName());
+      DlpJob response = dlpServiceClient.createDlpJob(createDlpJobRequest);
+
+      // Get the current time.
+      long startTime = System.currentTimeMillis();
+
+      // Check if the job state is DONE.
+      while (response.getState() != DlpJob.JobState.DONE) {
+        // Sleep for 30 second.
+        Thread.sleep(30000);
+
+        // Get the updated job status.
+        response = dlpServiceClient.getDlpJob(response.getName());
+
+        // Check if the timeout duration has exceeded.
+        long elapsedTime = System.currentTimeMillis() - startTime;
+        if (TimeUnit.MILLISECONDS.toMinutes(elapsedTime) >= TIMEOUT_MINUTES) {
+          System.out.println("Job did not complete within 15 minutes.");
```

Comment:
The message should use the value of `TIMEOUT_MINUTES` instead of 15, in case it is adjusted later.

Also, we should consider changing this instance too: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/91045acbd83ce5df93acd07894514dca26c69930/dlp/snippets/src/main/java/dlp/snippets/DeidentifyCloudStorage.java#L180